stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/adapter/view/internal/LinkAttachmentView.kt

```diff
@@ -66,15 +96,31 @@ internal class LinkAttachmentView : FrameLayout {
             binding.descriptionTextView.isVisible = false
         }
         style.textStyleLinkDescription.apply(binding.descriptionTextView)
+    }
 
+    /**
+     * Sets up the style for the link label text and displays it
+     * if it exists.
+     *
+     * @param attachment The attachment used to obtain the label.
+     * @param style The style which contains the label text style that
+     * will be applied.
+     */
+    private fun showLabel(attachment: Attachment, style: MessageListItemStyle) {
         val label = attachment.authorName
         if (label != null) {
             binding.labelContainer.isVisible = true
             binding.labelTextView.text = label.replaceFirstChar(Char::uppercase)
         } else {
             binding.labelContainer.isVisible = false
         }
+        style.textStyleLinkLabel.apply(binding.labelTextView)
```

Comment:
We seem to have missed applying this previously.

Reply:
I think this might be better:
`binding.labelTextView.setTextStyle(style.textStyleLinkLabel)`

Reply:
Valid suggestion, a scoping function here seems very unnecessary.
Changes here and in all of the other places we've used this style previously.